dataactvalidator/validation_handlers/validator.py

```diff
@@ -231,8 +231,7 @@ def validate(record,rules,csvSchema,fileType,interfaces):
         for rule in multiFieldRules:
             if not Validator.evaluateRule(record,rule,None,interfaces,record):
                 recordFailed = True
-                failedRules.append(["MultiField", "".join(["Failed rule: ",str(rule.description)]), Validator.getMultiValues(rule, record, interfaces)])
-
+                failedRules.append(["MultiField", "".join(["Failed rule: ",str(rule.rule_type.name)]), Validator.getMultiValues(rule, record, interfaces)])
```

Comment:
I don't agree with this change, the description is intended to be a human-readable explanation of the rule that failed, replacing that with just the type of rule that failed will not give users enough information in the error report to fix their files.